tests/e2e/tests/regression/browser/add-keys.e2e.ts

```diff
@@ -24,7 +24,8 @@ fixture `Different JSON types creation`
         await cliPage.sendCommandInCli(commandString);
         await deleteDatabase(ossStandaloneConfig.databaseName);
     });
-test('Verify that user can create different types(string, number, null, array, boolean) of JSON', async t => {
+// skipped due the issue https://redislabs.atlassian.net/browse/RI-2966
+test.skip('Verify that user can create different types(string, number, null, array, boolean) of JSON', async t => {
```

Comment:
Issue was already fixed, could you please validate?

Reply:
done